frontend/internals/webpack/webpack.common.config.js

```diff
@@ -18,7 +19,6 @@ module.exports = (env, options) => ({
 	output: {
 		path: PATHS.DIST_DIR,
 		filename: '[name].[chunkhash].js', 
-		pathinfo: true,
```

Comment:
The property `pathinfo` refers to weather to include comments in the final bundle.
I removed it because it defaults to `true` for development (so no difference in development) and to `false` in deployment.

I decided to remove because webpack guidelines reccommended it.
>**Warning**
  _While the data these comments can provide is useful during development when reading the generated code, it should not be used in production_
> \- [webpack website](https://webpack.js.org/configuration/output/#outputpathinfo)